app/views/categories/new.html.erb

```diff
@@ -3,11 +3,13 @@
 <div class='form up'>
 <h3 class="cat-header"> Add New Category </h3>
 <%= form_with model:@category, url: categories_path do |form| %>
-<div class='form-inputs'>
-    <div class="input-field col s6">
-      <%= form.text_field :name, class: "validate", id: "name" %>
-      <%= form.label :name %>
-    </div>
+<div class='form-inputs'>   
+    <div class="row">
+      <div class="input-field col s12">
+        <input id="category_name" type="text" value="" class="validate" name="category[name]" placeholder="Name" required autofocus>
+        <label for="category_name"></label>
```

Comment:
- [ ] [OPTIONAL] Great job @Rachelwebdev 👍🏽 , I would like to recommend that you place the form label before the form input field because at the moment the label is `NAME` is standing in as a placeholder in the input field that I need to click before the `NAME` to be able to type in a name. If a user clicks on the name directly which looks like a placeholder, such a user will not be able to type.

<img width="369" alt="Screenshot 2023-06-03 at 13 24 42" src="https://github.com/Rachelwebdev/budget-app/assets/58692590/f794bd5c-19fa-462e-8ee4-c82b712c5430">

Reply:
Thank you for this feedback, I will work on this and make the optional changes 👍